origami.json

```diff
@@ -1,23 +1,24 @@
 {
 		"description": "Component description",
+		"keywords": "component keywords for the registry search",
```

Comment:
Ok, @mariabg do you want to test that this works in the Registry when doing a keyword search, or we can just remove the `,` from the string before outputting to the attribute.